superset/assets/javascripts/components/StackTraceMessage.jsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Alert, Collapse } from 'react-bootstrap';
+
+const propTypes = {
+  message: PropTypes.string,
+  queryResponse: PropTypes.object,
+  showStackTrace: PropTypes.bool,
+};
+const defaultProps = {
+  showStackTrace: false,
+};
+
+class StackTraceMessage extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showStackTrace: props.showStackTrace,
+    };
+  }
+
+  hasTrace() {
+    return this.props.queryResponse && this.props.queryResponse.stacktrace;
+  }
+
+  render() {
+    const msg = (
+      <div>
+        <p
```

Comment:
we have used this a couple of places, could create an `HTML` component for it (nbd)

Reply:
somehow I didn't see this was used in other place...